src/components/app/data/hooks/useEnterpriseCourseEnrollments.js

```diff
@@ -51,7 +51,10 @@ export default function useEnterpriseCourseEnrollments(queryOptions = {}) {
     bffQueryOptions: {
       ...queryOptions,
       select: (data) => {
-        const transformedData = data.enterpriseCourseEnrollments.map(transformCourseEnrollment);
+        const transformedData = {
+          enrollments: data.enterpriseCourseEnrollments,
+          enrollmentsByStatus: data.allEnrollmentsByStatus,
```

Comment:
[inform] Right now, the BFF returns only realized enterprise course enrollments in `allEnrollmentsByStatus`; however, it would eventually be extended to contain the assignments/requests , too. When this happens, the below queries to get the assignments/requests data would be disabled when on a BFF-enabled page route (i.e., when sourced from the BFF API response instead of the current data sources).